boards/ti/am243x_evm/am243x_evm_am2434_r5f0_0_defconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Texas Instruments Incorporated
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
+
+# Serial Driver
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable Mailbox
+CONFIG_MBOX=y
```

Comment:
You don't need to manually enable MBOX here, it should be enabled by projects that use it. For instance the [OpenAMP examples](https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/ipc/openamp/prj.conf#L2) enable it in their prj.conf (although in that case they enable `CONFIG_IPM` which then enables `CONFIG_MBOX`, but you get the idea).